packages/messaging/src/controllers/sw-types.ts

```diff
@@ -63,7 +63,7 @@ interface ClientQueryOptions {
 }
 
 interface WindowClient extends Client {
-  readonly ancestorOrigins: ReadonlyArray<string>;
+  readonly ancestorOrigins: readonly string[];
```

Comment:
Nice. Is this checked by ESLint?

Reply:
I didn't touch it, so it must be auto fixed.

Reply:
Cool. It's a TS 3.4 feature, good to see ESLint being quick to update stuff like this. It has a lot of extra rules that we didn't have in TSLint as well. Initially I had my doubts, but I'm sold on ESLint. :smile: :+1: